### ISSUE TITLE
PLY File Not Rendering in Scene Despite No Errors – Possible Material Issue

### ISSUE THREAD

**Issue by Gemstone-t** [mkkellogg/GaussianSplats3D#395]
### Discussed in https://github.com/mkkellogg/GaussianSplats3D/discussions/394

<div type='discussions-op-text'>

<sup>Originally posted by **Gemstone-t** December 16, 2024</sup>
I am encountering an issue when trying to load a PLY file into my scene. The file loads without any errors in the console, but nothing appears in the scene. I have confirmed that the PLY file can be successfully opened and viewed using 3D Viewer, so the file itself seems fine.

I suspect the issue might be related to the lack of material or color information in the PLY file, which could be why nothing is visible.

Could someone please help me resolve this issue? I would appreciate any guidance as soon as possible.</div>

**Comment by Gemstone-t** [comment]
I use the ply files from "[https://www.artec3d.com/3d-models/ply](https://www.artec3d.com/3d-models/ply)"

**Comment by Gemstone-t** [comment]
So, is it possible to show this file using Gaussian Splat 3D?
I am currently loading and displaying the file with PLYLoader in Three.js, where I apply the material to the PLY geometry. Is there a way to set the material in Gaussian Splat 3D in a similar manner?

**Comment by seppestaes** [comment]
INRIAV2PlyParser has 'x', 'y' and 'z', but requires the other props to be present (INRIAV2PlyParser.js) as expected/needed :) The viewer offers a toggle between splats/points, still the point representation is based on splat props.

**Comment by Gemstone-t** [comment]
Is it impossible to show like these PLY files by using GaussianSplats3D.Viewer?

**Comment by seppestaes** [comment]
Yes, I believe so. Out of curiosity, why use a splat viewer for .ply files that only contain point data? What would be the added value in comparison with PLYLoader?

**Comment by Gemstone-t** [comment]
Thank you for your help.

**Comment by seppestaes** [comment]
I'm just an enthousiast of this repo :)

**Comment by mkkellogg** [comment]
Closing this one, as it sounds like the issue is resolved.
